controllers/ibu_controller.go

```diff
@@ -65,6 +65,28 @@ type ImageBasedUpgradeReconciler struct {
 	ManifestClient  *clusterinfo.InfoClient
 	OstreeClient    ostreeclient.IClient
 	Ops             ops.Ops
+	PrepConfig      *PrepConfig
```

Comment:
Not to block this PR:
- The type, maybe even the variable itself, is not specific to the Prep stage.
- Config feels like a wrong name for this? Task/job/step all sounds a little better?

Reply:
I didn't like the name `config` either, I will change it to `task` as job is a k8s jargon.